chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/results/CertificationIdResults.java

```diff
@@ -52,8 +51,10 @@ public void setProductId(final Long id) {
     private String ehrCertificationId;
     private Map<String, Integer> metCounts;
     private Map<String, Integer> metPercentages;
-    private ArrayList<String> missingAnd;
-    private List<ArrayList<String>> missingOr;
+    protected ArrayList<String> missingAnd = new ArrayList<String>();
+    protected List<ArrayList<String>> missingOr = new ArrayList<ArrayList<String>>();
```

Comment:
Checkstyle tends to complain about protected stuff. If you think it's warranted it's probably okay to leave it this way, but if you can make this private and add getters/setters it'll help with our linting counts